zenodo/modules/stats/registrations.py

```diff
@@ -167,12 +168,13 @@ def register_queries():
                     is_parent='is_parent'
                 ),
                 query_modifiers=[
-                    lambda query, _: query.filter('term', is_parent=True)
+                    lambda query, **_: query.filter('term', is_parent=True)
```

Comment:
The actual call looked something like:

```python
for modifier in self.query_modifiers:
    self.agg_query = modifier(self.agg_query, **kwargs)
```

so "kwargs" has to be star-expanded